homeassistant/components/tuya/cover.py

```diff
@@ -116,13 +117,15 @@ class TuyaCoverEntityDescription(CoverEntityDescription):
         TuyaCoverEntityDescription(
             key=DPCode.CONTROL,
             translation_key="curtain",
+            current_instruction=DPCode.CONTROL,
```

Comment:
If the current instruction is always the same as the key, why add another property to the entity description?

Reply:
it's always the same for the `clkg` family.
For other devices it's `NONE`, as I didn't have such devices and I can't predict how they behave.

Reply:
Sorry, you missed the point, you added the current_instruction... but that is already handled because of the key.

Reply:
replaced duplicate value by a boolean flag to enable the behavior.